### PR TITLE
Set IBM Cloud specific vendor and icons

### PR DESCRIPTION
**Pull request by jaywcarman**
See https://github.com/ManageIQ/manageiq-providers-ibm_cloud/issues/243

Use IBM PowerVC specific vendor and icon names to differentiate from
other IBM providers (IBM PowerVC).

Depends on
- https://github.com/ManageIQ/manageiq/pull/21333
- https://github.com/ManageIQ/manageiq-decorators/pull/50
- https://github.com/ManageIQ/manageiq-schema/pull/587

### CODE REVIEW

**Comment by jaywcarman** [comment]
@miq-bot cross-repo-tests ManageIQ/manageiq#21333

File: app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb [hunk 1]
```diff
@@ -42,7 +42,7 @@ def pvm_instances
         :flavor            => persister.flavors.lazy_find(instance.sys_type),
         :location          => _("unknown"),
         :name              => instance.server_name,
-        :vendor            => "ibm",
+        :vendor            => "ibm_cloud",
```
Comment (agrare) [inline app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb:45]: If we're going to change the vendor of existing vm records we're going to need a schema data migration to change existing db records that have `:vendor => "ibm"`

Comment (jaywcarman) [reply app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb:45]: Good catch. I'll work on a patch today.

Comment (jaywcarman) [reply app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb:45]: https://github.com/ManageIQ/manageiq-schema/pull/587

**Comment by agrare** [comment]
:+1: looks good

**Review by agrare — APPROVED** [review]

**Pull request merged**